flow/error_definitions.h

```diff
@@ -158,6 +158,10 @@ ERROR( tag_too_long, 2110, "Tag set on transaction is too long" )
 ERROR( too_many_tag_throttles, 2111, "Too many tag throttles have been created" )
 ERROR( special_keys_cross_module_read, 2112, "Special key space range read crosses modules. Refer to the `special_key_space_relaxed' transaction option for more details." )
 ERROR( special_keys_no_module_found, 2113, "Special key space range read does not intersect a module. Refer to the `special_key_space_relaxed' transaction option for more details." )
+ERROR( special_keys_write_disabled, 2114, "Special Key space is not allowed to write by default. Refer to the `special_key_space_enable_writes` transaction option for more details." )
+ERROR( special_keys_no_write_module_found, 2115, "Special key space key or keyrange in set or clear does not intersect a module" )
+ERROR( special_keys_cross_module_clear, 2116, "Special key space clear crosses modules" )
+ERROR( special_keys_api_failure, 2117, "Api call through special keys failed. For more information, call get(\"\xff\xff/error_message\") to get a json string of the error message." )
```

Comment:
We probably want this error message to be valid utf-8, so let's change the literal \xff\xff's

Reply:
Done. Thanks.